docs.md

```diff
@@ -171,3 +173,75 @@ As you can tell after running the code and logging out the method, we get the fo
 ```
 
 Alright, so that's the `Account` class in a nutshell! If you have any things you would like to add please send a pull request or an issue so we can fix it!
+
+### Validator
+
+In this section we are going to talk about the base Validator node that both the Primary Validator and the Confirmation Validator inherit from. We will be going over the basic usage of any Validator, which is to validate transactions, but to learn more about where they deviate you can refer to their respective sections.  For now in this section, and for the sake of shorthand we will refer to both types as just "Validators", but with code examples for each.
+
+### Creating Validators
+
+To create a Validator we must pass in the url of the Validator you wish to interact with, into a constructor.
+
+```ts
+// create object with access to the API and basic functions of a Validator.
+const confirmationValidator = new ConfirmationValidator("http://157.230.10.237");
+const primaryValidator = new PrimaryValidator("http://157.230.75.212");
+
+// Get the current config data for validators, must be in asynchronous function.
+async function getValidatorConfigs() {
+  let configs = {
```

Comment:
We might be better off with just using top-level await as the library _is_ built with TypeScript.  Anyone who is writing in JavaScript with a library for this complex of a use would understand that they would need an asynchronous IIFE.  Also, it adds needless complexity to our code examples.  Perhaps it would be good if we added a blockquote with something like this:
> In our examples, we will be using async/await instead of the .then format of [handling Promises](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Using_promises).  It is important to remember that you would need to have an asynchronous function (possibly [an IIFE](https://developer.mozilla.org/en-US/docs/Glossary/IIFE)) to use top-level async/await in your regular CommonJS and browser JavaScript.  However, it is [available in TypeScript](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html#top-level-await) and [V8 modules](https://v8.dev/features/top-level-await#:~:text=Top%2Dlevel%20await%20enables%20developers,they%20start%20evaluating%20their%20body.).

Reply:
Yeah I was wondering If it was too much detail, since import and require are also implicit.  I'll update that and leave it out for future examples.

Reply:
Yeah, the `import` and `export` statements are sort of implicit, I'll post a discussion question in the slack for if we should include them in our examples👍